includes/class-scd-ext-utilities.php

```diff
@@ -99,4 +99,44 @@ public function get_course_users( $course_id ){
 		return $course_users;
 
 	}// end get_course_users
+
+    /**
+     *  check_for_translation() function
+     *
+     *  Handles which message to show users
+     *  when the message is both set in a
+     *  translation, as well as under Sensei ->
+     *  Settings -> Content Drip.
+     *
+     *	@param str $default_message
+     *  @param str $settings_field
+     *  @return str $message
+     */
+
+    public function check_for_translation($default_message, $settings_field) {
```

Comment:
Not assigning to an intermediate `$message` var makes this a cleaner read. Maybe something like this:

```
    public function check_for_translation( $default_message, $settings_field ) {
 
         $possible_translation = __( $default_message, 'sensei-content-drip' ) ;
 
         // Return the translation if present
         if ($possible_translation != $default_message ) {
             return $possible_translation;
         }

         // If not, check if "Sensei -> Settings -> Content Drip" is set and return that
         // If that is not set either, return the default English string.
         $settings_message =  Sensei_Content_Drip()->settings->get_setting( $settings_field );
         return ( empty( $settings_message ) ) ? $default_message : $settings_message;
     } // end check_for_translation()
```